plugins/MakerbotWriter/MakerbotWriter.py

```diff
@@ -0,0 +1,270 @@
+# Copyright (c) 2023 UltiMaker
+# Cura is released under the terms of the LGPLv3 or higher.
+
+from io import StringIO, BufferedIOBase
+import json
+from typing import cast, List, Optional, Dict
+from zipfile import BadZipFile, ZipFile, ZIP_DEFLATED
+import pyDulcificum as du
+
+from PyQt6.QtCore import QBuffer
+
+from UM.Logger import Logger
+from UM.Math.AxisAlignedBox import AxisAlignedBox
+from UM.Mesh.MeshWriter import MeshWriter
+from UM.PluginRegistry import PluginRegistry
+from UM.Scene.SceneNode import SceneNode
+from UM.Scene.Iterator.DepthFirstIterator import DepthFirstIterator
+from UM.i18n import i18nCatalog
+
+from cura.CuraApplication import CuraApplication
+from cura.Snapshot import Snapshot
+from cura.Utils.Threading import call_on_qt_thread
+from cura.CuraVersion import ConanInstalls
+
+catalog = i18nCatalog("cura")
+
+
+class MakerbotWriter(MeshWriter):
+    """A file writer that writes '.makerbot' files."""
+
+    def __init__(self) -> None:
+        super().__init__(add_to_recent_files=False)
+        Logger.info(f"Using PyDulcificum: {du.__version__}")
+
```

Comment:
```suggestion
        MimeTypeDatabase.addMimeType(
            MimeType(
                name = "application/x-makerbot",
                comment = "UltiMaker Format Package",
                suffixes = ["makerbot"]
            )
        )
```

Could this be added inside __init__. Mimetype is needed while uploading the file to the cloud

Reply:
https://github.com/Ultimaker/Cura/pull/17147/commits/2660933d6ea9c29b9d76f96d2c07f335481dd546